pkg/plugins/diff/service.go

```diff
@@ -19,6 +18,8 @@ import (
 
 var (
 	ErrTableNotFound = errors.New("table not found")
+	ErrLoadingPlugin = errors.New("couldn't load diff plugin")
```

Comment:
```suggestion
	ErrLoadingPlugin = errors.New("failed to load diff plugin")
```
and perhaps it should wrap `ErrFailedDiff`, to make it easier to manage both of them.